modules/standard/IO.chpl

```diff
@@ -2310,8 +2319,15 @@ proc openfp(fp: _file, hints=ioHintSet.empty):file throws {
   return openfpHelper(fp, hints);
 }
 
-private proc openfpHelper(fp: c_FILE, hints=ioHintSet.empty,
+private proc openfpHelper(fp: ?t, hints=ioHintSet.empty,
                           style:iostyleInternal = defaultIOStyleInternal()):file throws {
+
+  // check if the user passed in a c_FILE rather than a c_ptr(c_FILE)
```

Comment:
Can you also add a comment to remove the type query (and replace with `c_ptr(c_FILE)`) when this is removed